Módulos

In [1]:
# Manipulação de dados
import numpy as np 
import pandas as pd
import datetime as dt

# Visualização de Dados
import matplotlib.pyplot as plt
import seaborn as sns 

Carregando Dados

In [2]:
dados = pd.read_csv('dataset.csv')

Shape

In [3]:
dados.shape

(9700, 11)

Amostra dos dados

In [5]:
dados.head(3)

ID_Pedido Data_Pedido ID_Cliente   Segmento           Pais  \
0  CA-2017-152156  08/11/2017   CG-12520   Consumer  United States   
1  CA-2017-152156  08/11/2017   CG-12520   Consumer  United States   
2  CA-2017-138688  12/06/2017   DV-13045  Corporate  United States   

        Cidade      Estado       ID_Produto        Categoria SubCategoria  \
0    Henderson    Kentucky  FUR-BO-10001798        Furniture    Bookcases   
1    Henderson    Kentucky  FUR-CH-10000454        Furniture       Chairs   
2  Los Angeles  California  OFF-LA-10000240  Office Supplies       Labels   

   Valor_Venda  
0       261.96  
1       731.94  
2        14.62

# Perguntas de Negócios

## Pergunta de Negócio 1:
    Qual Cidade com Maior Valor de Venda de Produtos da Categoria 'Office Supplies'?

## Pergunta de Negócio 2:
    Qual o Total de Vendas Por Data do Pedido?Demonstre o resultado através de um gráfico de barras.

## Pergunta de Negócio 3:
    Qual o Total de Vendas por Estado?Demonstre o resultado através de um gráfico de barras.

## Pergunta de Negócio 4:
    Quais São as 10 Cidades com Maior Total de Vendas?Demonstre o resultado através de um gráfico de barras.

## Pergunta de Negócio 5:
    Qual Segmento Teve o Maior Total de Vendas?Demonstre o resultado através de um gráfico de pizza.

## Pergunta de Negócio 6 (Desafio Nível Baby):
    Qual o Total de Vendas Por Segmento e Por Ano?

## Pergunta de Negócio 7 (Desafio Nível Júnior):
    Os  gestores  da  empresa  estão  considerando conceder  diferentes  faixas  de  descontos  e gostariam de fazer uma simulação com base na regra abaixo:

    Se o Valor_Venda for maior que 1000 recebe 15% de desconto.
    Se o Valor_Venda for menor que 1000 recebe 10% de desconto.

Quantas Vendas Receberiam 15% de Desconto?

## Pergunta de Negócio 8 (Desafio Nível Master):
    Considere  Que  a  Empresa  Decida  Conceder  o  Desconto  de  15%  do  Item  Anterior.  Qual Seria a Média do Valor de Venda Antes e Depois do Desconto?

## Pergunta de Negócio 9 (Desafio Nível Master Ninja):
    Qual o Média de Vendas Por Segmento, Por Ano e Por Mês?Demonstre o resultado através de gráfico de linha.

## Pergunta de Negócio 10 (Desafio Nível Master Ninja das Galáxias):
    Qual o Total de Vendas Por Categoria e SubCategoria, Considerando Somente as Top 12 SubCategorias?Demonstre tudo através de um único gráfico.